x-pack/plugins/siem/public/pages/overview/translations.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const NO_FILEBEAT_INDICES = i18n.translate(
+  'xpack.siem.overview.network.noFilebeatIndicies',
+  {
+    defaultMessage: "Looks like you don't have any Filebeat and Auditbeat indices.",
+  }
+);
+
+export const LETS_ADD_SOME = i18n.translate('xpack.siem.overview.letsAddSome.description', {
```

Comment:
Need to match [i18n key naming conventions](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#naming-convention)

Reply:
As discussed we will do that in round 2 PR or later.